modules/hub/src/util/logging.ts

```diff
@@ -93,7 +96,7 @@ export class SCLogger {
 
     const c = console as any
 
-    if (process.env.NODE_ENV !== 'production' || !this.logdna) {
+    if ((process.env.NODE_ENV !== 'production' || !this.logdna) && levelno >= logLevelLimit) {
```

Comment:
Seems a little clunky to just compare to a hardcoded limit, not sure how to use hub's config. Maybe just use process.env?

Reply:
process.env is probably fine for this.

Reply:
Yes